source/style-guide/quickstart.txt

```diff
@@ -86,12 +86,11 @@ the following ones to help you boost the effectiveness of your writing.
      - :ref:`use-correct-punctuation`
 
        :ref:`punctuation`
-   * - **Use sentence-style capitalization for all titles and
+   * - **Use headline-style capitalization for most titles and
```

Comment:
[question]

Since we're now using "most" instead of "all", are there any examples we can provide where writers should not use headline-style capitalization? One example I can think of might be if the heading contains an operator name, and the operator begins with a lower-case letter.

Reply:
Actually I think it's fine to use "all" instead of "most" here, since I'm realizing the exceptions (like the operator name) would just be one word. The headline-style guidelines include lowercasing "words that always begin with a lowercase letter", so a title with one lowercase word would still be following the headline rules